google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -138,15 +139,23 @@ def create_finding(self, finding, source_id=None, finding_id=None):
                 '{}/findings/{}'.format(source_id, finding_id),
                 finding
             )
-            LOGGER.debug('Created finding response: %s',
+            LOGGER.debug('Successfully created finding response: %s',
                          response)
             return response
         except (errors.HttpError, HttpLib2Error) as e:
-            LOGGER.exception(
-                'Unable to create CSCC finding: Resource: %s', finding)
-            violation_data = (
-                finding.get('source_properties').get('violation_data'))
-            raise api_errors.ApiExecutionError(violation_data, e)
+            raw_error = e.args[1]
+            error = raw_error.decode('utf-8')
+            formatted_error = json.loads(error)
+            error_msg = formatted_error['error']['message']
+            if error_msg == 'Requested entity already exists':
```

Comment:
Checking on a text output seems to be fragile, as this can stop working if the text changes.  What do you think about checking the error code `409`?  This way, it will be more robust to text changes. Is the error code in the exception?

Reply:
Yes, error code is also part of the raw error. Updated code to check for error code instead.